src/components/Clickable/Clickable.scss

```diff
@@ -0,0 +1,20 @@
+@import "../../styles/themes.scss";
+@import "../../styles/typography.scss";
+@import "../../styles/states.scss";
+
+.monday-style-clickable {
+  @include focus-style;
+  max-width: fit-content;
```

Comment:
not sure about the fit content. is it easier this way or not needed?

Reply:
i don't think it is needed